src/docker-generate-binaries.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/sh
+set -ex
+# maven plugin version
+VER=$1
+# output directory of binaries
+DEST=$2
+docker build --build-arg VERSION=$VER -o $DEST -f Dockerfile.linux .
```

Comment:
```suggestion
export DOCKER_BUILDKIT=1
docker build --build-arg VERSION=$VER -o $DEST -f Dockerfile.linux .
```

?

Reply:
ahh, was this why you were producing empty jars? Or is this a separate issue?

Reply:
Set docker buildkit flag in edf248c

Reply:
https://repo.jenkins-ci.org/incrementals/io/jenkins/plugins/lib-durable-task/1.0-rc34.edf248cdcad1/lib-durable-task-1.0-rc34.edf248cdcad1.jar indeed seems to contain four binaries as expected.